test/test_mps.py

```diff
@@ -5901,8 +5915,8 @@ def helper(shape, padding, op, value=0):
         helper((2, 1, 6), 3, nn.ReplicationPad1d)
         # Constant Pad 1D
         helper((2, 3, 4), 2, nn.ConstantPad1d)
-        # Constant Pad 1D with single dimension input
-        helper((16), (1, 2), nn.ConstantPad1d)
+        # Constant Pad with single dimension input
+        helper_functional((16), (1, 2), 'constant')
```

Comment:
`nn.functional.pad` allows **constant** padding for arbitrary dimensions (see [docs](https://pytorch.org/docs/stable/generated/torch.nn.functional.pad.html)), however, `nn.ConstantPad{1,2,3}d` only allow 2/3D, 3/4D and 4/5D inputs respectively based on the [docs (e.g. here for 1D)](https://pytorch.org/docs/stable/generated/torch.nn.ConstantPad1d.html?highlight=constantpad1d#torch.nn.ConstantPad1d). Changing the cases where input shapes are invalid according the docs to test the functional variant here